packages/shared-influx-pieces/src/listeners/resourceAnalyticsSync.ts

```diff
@@ -0,0 +1,38 @@
+import type { PieceContext } from '@skyra/http-framework';
+import { Point } from '@skyra/influx-utilities';
+import { AnalyticsListener } from 'lib/AnalyticsListener.js';
+import { Points, Tags, Actions } from 'lib/AnalyticsSchema.js';
+import { cpus } from 'os';
+
+export class UserAnalyticsEvent extends AnalyticsListener {
+	public constructor(context: PieceContext, options: AnalyticsListener) {
+		super(context, { ...options, event: 'resourceAnalyticsSync' });
+	}
+
+	public run() {
+		this.writePoints([
+			this.syncPerCoreLoad(), //
+			this.syncMem()
```

Comment:
The two methods don't sync, they read the resources and create the points.

In fact I think it may be just as efficient to do `this.writePoint` in each method to avoid creating an array, it'll also make the method prettier:

```typescript
this.writeCoreUsagePoint();
this.writeMemoryUsagePoint();
this.flush();
```